idd/Energy+.idd.in

```diff
@@ -52924,6 +53042,368 @@ AirConditioner:VariableRefrigerantFlow,
+  N25, \field Ratio of Compressor Size to Total Compressor Capacity
+       \type real
+       \units W/W
+       \default 0.5
+       \note Enter the ratio of the first stage compressor to total compressor capacity
+       \note All other compressors are assumed to be equally sized. This inputs is used
+       \note only for crankcase heater calculations
+  N26, \field Maximum Outdoor Dry-Bulb Temperature for Crankcase Heater
+       \type real
+       \units C
+       \default 5.0
+       \note Enter the maximum outdoor temperature above which the crankcase heaters are disabled
+  A8 , \field Defrost Strategy
+       \type choice
+       \key ReverseCycle
+       \key Resistive
+       \default Resistive
+       \note Select a defrost strategy. 
+       \note Reverse cycle reverses the operating mode from heating to 
+       \note cooling to melt frost formation on the condenser coil
+       \note The resistive strategy uses a resistive heater
+       \note to melt the frost.
+  A9 , \field Defrost Control
+       \type choice
+       \key Timed
+       \key OnDemand
+       \default Timed
+       \note Choose a defrost control type
+       \note Either use a fixed Timed defrost period or select
+       \note OnDemand to defrost only when necessary
+  A10, \field Defrost Energy Input Ratio Modifier Function of Temperature Curve Name
+       \type object-list
+       \object-list LinearCurves
+       \object-list QuadraticCubicCurves
+       \object-list UniVariateTables
+       \note A valid performance curve must be used if ReverseCycle defrost strategy is selected
+  N27, \field Defrost Time Period Fraction
+       \type real
+       \minimum 0.0
+       \default 0.058333
+       \note Fraction of time in defrost mode
+       \note Only applicable if timed defrost control is specified
+  N28, \field Resistive Defrost Heater Capacity
+       \type real
+       \minimum 0.0
+       \default 0.0
+       \autosizable
+       \units W
+       \note Enter the size of the resistive defrost heating element
+       \note Only applicable if resistive defrost strategy is specified
+       \ip-units W
+  N29, \field Maximum Outdoor Dry-bulb Temperature for Defrost Operation
+       \type real
+       \units C
+       \default 5.0
+       \note Enter the maximum outdoor temperature above which the defrost operation is disabled
+  N30, \field Compressor maximum delta Pressure
+       \type real 
+       \units Pa
+       \default 4500000.0
+       \minimum 0.0
+       \maximum 50000000.0
+  N31, \field Number of Compressor Loading Index Entries
+       \required-field
+       \type integer
+       \default 2
+       \minimum 2
+       \maximum 9
+       \note First index represents minimal capacity operation 
+       \note Last index represents full capacity operation 
+  N32, \field Compressor Speed at Loading Index 1 
+       \type real
+       \note Minimum compressor speed 
+       \note unit in Rev/Sec	
+       \minimum> 0   
```

Comment:
Units should be declared as \units rev/sec (similar to the existing rev/min).  Also add to the units list at the top of the idd.  Same for all speed fields.